tcf_website/static/reviews/review_stats.css

```diff
@@ -0,0 +1,18 @@
+.large-stats-container {
+    padding-left: 15px;
+    padding-right: 15px;
+}
+
+.large-stats {
+    padding-top: 15px;
+    padding-bottom: 10px;
+    background-color: white;
+    text-align: center;
+    display: block;
+    margin-left: 1px;
+    width: 33.33%

```

Comment:
Do you need the `width: 33.33%` if you're already using bootstrap columns?

Reply:
I'm not really good at frontend, but the `width` was there in the previous code and it looks like this without the line:
![image](https://user-images.githubusercontent.com/7676354/100983398-9c5d8400-358c-11eb-8ffe-623cc8d00586.png)
Seems like it was there for a reason in the first place?

Reply:
ehhh that's strange 
lol just leave it then